tests/functional/test_dependencies.py

```diff
@@ -137,9 +137,9 @@ def test_decode_dependency_with_config_override(project):
 def test_uri_map(project_with_dependency_config):
     actual = project_with_dependency_config.dependencies.uri_map
     here = Path(__file__).parent
-    expected = f"file://{here}/data/projects/LongContractsFolder"
+    expected = Path(f"file://{here}/data/projects/LongContractsFolder")
```

Comment:
how come this is wrapped in Path? Windows? Maybe add a comment.
Note: this the str is a file URI (starting with `file://`) so it is not typical to wrap, but I think I see why you did it

Reply:
Yes, it was failing for me on Windows because the path separator character. Will add a comment